test/parse-args.js

```diff
@@ -39,6 +67,87 @@ describe('parse-args', () => {
   })
 
   describe('--config', () => {
+    it('c8 process should throw an error message if an invalid configuration file name is passed', () => {
+      const invalidConfig = './fixtures/config/.c8.config.py'
+      const loadInvalidConfigFile = (file, callBack) => {
+        try {
+          callBack(file)
+          assert.equal(true, false, 'Invalid configuration file loaded')
+        } catch (error) {
+          assert.equal(true, true, `${error}`)
+        }
+      }
+
+      loadInvalidConfigFile(invalidConfig, (file) => {
+        loadConfigFile(file)
+      })
+    })
+
+    it('config directory should contain all variations of the config file naming convention', () => {
+      let count = 0
+      const fileMessages = []
+      const configFileList = getConfigFileNames()
+      configFileList.forEach((file) => {
+        const fullPath = './test/fixtures/config/' + file
+        if (existsSync(fullPath)) {
+          count++
+        } else {
+          fileMessages.push(`Missing ${file} from ./test/fixtures/config directory`)
+        }
+      })
+
+      if (count === configFileList.length) {
+        assert.equal(count, configFileList.length)
+      } else {
+        const msg = fileMessages.join(' \n      ')
+        assert.equal(fileMessages.length, 0, msg)
+      }
+    })
+
+    const filePath = './fixtures/config/'
+    describe('c8 variations of config file', () => {
+      describe('should be able to read config files with .json, .yml, .yaml, .js, .cjs extensions', () => {
+        beforeEach(beforeTestReadingConfigFile)
+
+        const fileNameLineNumberMap = {
+          '.c8rc.json': 101,
+          '.c8rc.yml': 69,
+          '.c8rc.yaml': 10,
+          'c8.config.js': 47,
+          'c8.config.cjs': 51,
+          '.c8rc.js': 22,
+          '.c8rc.cjs': 32,
+          '.c8.config.js': 47,
+          '.c8.config.cjs': 45
+        }
+
+        testReadingConfigFile(fileNameLineNumberMap, filePath)
+
+        afterEach(afterTestReadingConfigFile)
+      })
+    })
+
+    describe('nyc variations of config file', () => {
+      describe('should be able to read config files with .json, .yml, .yaml, .js, .cjs extensions', () => {
+        beforeEach(beforeTestReadingConfigFile)
+
+        const fileNameLineNumberMap = {
+          '.nycrc': 51,
+          '.nycrc.json': 96,
+          '.nycrc.yml': 99,
+          '.nycrc.yaml': 98,
+          'nyc.config.js': 95,
+          'nyc.config.cjs': 94,
+          '.nyc.config.js': 85,
+          '.nyc.config.cjs': 71
+        }
+
+        testReadingConfigFile(fileNameLineNumberMap, filePath)
+
+        afterEach(afterTestReadingConfigFile)
+      })
+    })
```

Comment:
If I comment out this block of code, code coverage improves back to acceptable levels.

Current Coverage if this block is included.

```#/bin/bash
--------------------------|---------|----------|---------|---------|--------------------------------
File                      | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s              
--------------------------|---------|----------|---------|---------|--------------------------------
All files                 |   75.67 |     67.1 |   71.87 |   75.67 |                                
 bin                      |   86.04 |       60 |     100 |   86.04 |                                
  c8.js                   |   86.04 |       60 |     100 |   86.04 | 20,33-35,41-42                 
 lib                      |   79.97 |    69.23 |      84 |   79.97 |                                
  parse-args.js           |   98.52 |     90.9 |     100 |   98.52 | 165-166,187-188                
  report.js               |   65.42 |       60 |   71.42 |   65.42 | ...354,358-359,375-380,391-392 
  source-map-from-file.js |      88 |    53.84 |     100 |      88 | 48-49,59,74-76,91-93,95-97     
 lib/commands             |   41.96 |       50 |   16.66 |   41.96 |                                
  check-coverage.js       |   18.57 |      100 |       0 |   18.57 | 9-11,14-36,39-53,55-70         
  report.js               |   80.95 |       40 |      50 |   80.95 | 9-10,15-20                     
--------------------------|---------|----------|---------|---------|--------------------------------
ERROR: Coverage for lines (75.67%) does not meet global threshold (95%)
ERROR: Coverage for branches (67.1%) does not meet global threshold (82%)
ERROR: Coverage for statements (75.67%) does not meet global threshold (95%)
Error: Process completed with exit code 1.
```